proc/threads.c

```diff
@@ -1128,6 +1023,19 @@ static void _proc_threadDequeue(thread_t *t)
 }
 
 
+static void _proc_threadInsertSleeping(thread_t *t)
```

Comment:
Hmm, I'm not convinced that solves the problem: `lib_rbInsert` can only return not `EOK` if `tree->compare` (in this case `threads_sleepcmp`) returns 0 - which it doesn't. So we'd still insert thread with the same `wakeup`

Reply:
Right, I've reverted the change